scripts/run-once-scripts/promote-special-trial-judge.ts

```diff
@@ -1,3 +1,4 @@
+// TODO: Remove, or call update-judge.ts?
```

Comment:
Up to you, everything in run-once scripts is meant to be throw away. Feel free to delete or use for reference. It is code that is meant to be used.... once.